internal/installation/receipt/receipt_test.go

```diff
@@ -59,7 +60,11 @@ func TestLoad(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	testPluginReceipt := index.Receipt{Plugin: testPlugin}
+	testPluginReceipt := testutil.NewReceipt().WithPlugin(testPlugin).WithStatus(index.ReceiptStatus{
```

Comment:
(Couldn't comment there but) on line :33 in `TestStore()`, shouldn't we be using the new `testutil.NewReceipt()` as well?

Reply:
Ya I missed that test. I'll update it :+1: